### ISSUE TITLE
Split cli

### ISSUE THREAD

**Issue by ibatullin** [cornelius/inqlude#23]
Inqlude tool is used to managing manifests and installing libraries. I think the tool can be split into 2 tools:

- user tool that manage libraries
- admin tool that manage manifests

User tool example:
```
inqlude install <library> - installs library
inqlude uninstall <library> - removes library
inqlude download <library> - downloads library
inqlude list - shows all available libraries
inqlude scan - scans system
```

Admin tool example:
```
inqluder create <library> - creates manifest for library
inqluder kde release <library> - creates kde release
inqluder publish
```

**Comment by cornelius** [comment]
I agree with that. There actually is a client implementation of the user tool at https://projects.kde.org/projects/playground/sdk/inqlude-client. It's done in C++ and Qt, so this might be a bit more natural for Qt users.

The admin functionality could be moved to a separate tool. `inqluder` is a clever name :smile:. I was thinking about `inqlude-admin`.

**Comment by cornelius** [comment]
As a first step I would restructure the commands to separate client and admin functionality. The structure could look like this:

`client list` (was `list`)
`client show` (was `show`)
`client download` (was `download`)
`client install` (was `install`)
`client uninstall` (was `uninstall`)

`help`
`get_involved`

`kf create` (was `create_kde_frameworks`)
`kf release` (was `release_kde_frameworks`)

`view`
`create`
`review`
`system_scan`
`verify`
